### DISCUSSION TITLE
Missing justify content "normal"

### DISCUSSION THREAD

**Discussion by DavydeVries** [tailwindlabs/tailwindcss#10558] [Help] [answered]
Hi,

Currently, I'm using for viewport `sm` the class `sm:justify-center`. But I like to make the element full width on viewport `md`. The default value of `justify-content` is normally `justify-content: normal` (maybe differs on other browsers?). But there is no way to "reset" back to `normal` value.

Do you have suggestions to solve my issue, or is this value missing from the core?

Example:
Element not full width: https://play.tailwindcss.com/pj23KKG5cV
What I like to achieve/Expected behaviour: https://play.tailwindcss.com/oQ38fSUAsn

Kind regards,
Davy

_PS. I already tried to add `normal` to my config as an "extend" but didn't get it to work._

**Comment by wongjn** [comment, marked as answer]
Several solutions:

## Use an [arbitrary property class](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-properties) to apply `justify-content: normal`

```diff
-<div class="md:justify-normal flex gap-4 divide-dashed divide-brown-400 overflow-y-auto whitespace-nowrap p-4 text-center text-sm text-white sm:justify-center md:grid md:gap-0 md:divide-y md:whitespace-normal md:p-0 md:text-left md:text-lg">
+<div class="md:[justify-content:normal] flex gap-4 divide-dashed divide-brown-400 overflow-y-auto whitespace-nowrap p-4 text-center text-sm text-white sm:justify-center md:grid md:gap-0 md:divide-y md:whitespace-normal md:p-0 md:text-left md:text-lg">
```

## Use a `max-*` screen variant to apply `justify-content: center` for `sm` screen and lower

```diff
-<div class="md:justify-normal flex gap-4 divide-dashed divide-brown-400 overflow-y-auto whitespace-nowrap p-4 text-center text-sm text-white sm:justify-center md:grid md:gap-0 md:divide-y md:whitespace-normal md:p-0 md:text-left md:text-lg">
+<div class="flex gap-4 divide-dashed divide-brown-400 overflow-y-auto whitespace-nowrap p-4 text-center text-sm text-white max-sm:justify-center md:grid md:gap-0 md:divide-y md:whitespace-normal md:p-0 md:text-left md:text-lg">
```

## [Use a Tailwind plugin](https://tailwindcss.com/docs/plugins#static-utilities) to add a rule for `justify-content: normal`

```diff
   },
-  plugins: [],
+  plugins: [
+    require('tailwindcss/plugin')(({ addUtilities }) => {
+      addUtilities({
+        '.justify-normal': {
+          justifyContent: 'normal',
+        },
+      });
+    }),
+  ],
 }
```

## Use CSS to add a rule for `justify-content: normal`

```diff
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
 
+@layer utilities {
+  .justify-normal {
+    justify-content: normal;
+  }
+}
```

**Reply by DavydeVries** [reply]
Thank you so much, I went for the last option aka:
```
@layer utilities {
  .justify-normal {
    justify-content: normal;
  }
}
```

> _But I'm still convienced that this value should be in the core._
